pontoon/sync/migrations/0003_change_pontoon_sync_email.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.13 on 2020-08-28 13:59
+
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def update_sync_user_email(apps, schema_editor):
+    User = apps.get_model("auth", "User")
+    User.objects.filter(email="pontoon-sync@mozilla.org").update(email="pontoon-sync@example.com")
+    User.objects.filter(email="pontoon-gt@mozilla.com").update(email="pontoon-gt@example.com")
+    User.objects.filter(email="pontoon-mt@mozilla.com").update(email="pontoon-mt@example.com")
+
+
+def revert_sync_user_email(apps, schema_editor):
+    User = apps.get_model("auth", "User")
+    User.objects.filter(email="pontoon-sync@example.com").update(email="pontoon-sync@mozilla.org")
+    User.objects.filter(email="pontoon-gt@example.com").update(email="pontoon-gt@mozilla.com")
+    User.objects.filter(email="pontoon-tm@example.com").update(email="pontoon-tm@mozilla.com")
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('sync', '0002_change_pontoon_sync_email'),
+    ]
+
+    operations = [
+         migrations.RunPython(
+            code=update_sync_user_email, reverse_code=revert_sync_user_email,
+        ),
+    ]
```

Comment:
Given that you took care of all three users in the base migration file, you can remove this entire file.

Reply:
Delete the migration 0003_change_pontoon_sync_email.py ?

Reply:
Correct.